fhir-resource-tooling/pom.xml

```diff
@@ -72,6 +72,27 @@
 			<artifactId>slf4j-api</artifactId>
 		</dependency>
 
+		<dependency>
+			<groupId>com.ibm.cos</groupId>
+			<artifactId> ibm-cos-java-sdk-core</artifactId>
```

Comment:
Is this implementation tied specifically to IBM COS? I think we would want to use a generic S3 client to allow us to write to non-IBM object stores, especially with all the upcoming talk of supporting azure.

Reply:
Yeah, fair, I've updated the dependencies.